docs/parcel-api.md

```diff
@@ -0,0 +1,50 @@
+# Parcel API
+Most parcel methods will be called on the parcel itself, with the excpetion being around mounting.
+
+## mounting
+Both mount methods return a [parcel object](/docs/parcel-api.md#parcel-object). The parcel object contains all additional exposed methods.
+### mountParcel
+`application.mountParcel(parcelConfig)`. Each application is provided a mountParcel function.
```

Comment:
I think this might make more sense as `props.mountParcel()`, since mountParcel is a prop

Reply:
Additionally, this is missing the second argument to the function.

```js
props.mountParcel(parcelConfig, props)
```